pages/common/hg-commit.md

```diff
@@ -0,0 +1,27 @@
+# hg commit
+
+> Commit all staged or specified files to the repository.
+
+- Commit staged files to the repository:
+
+`hg commit`
+
+- Commit a specific file or directory:
+
+`hg commit {{path/to/file}}`
+
+- Commit with a specific message:
+
+`hg commit -m/--message {{message}}`
```

Comment:
I think I'd prefer only the long version of the flag here - and the same further down. Thoughts, @agnivade?